test/unit/domain/fido-credential.test.ts

```diff
@@ -0,0 +1,111 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the
+ Apache License, Version 2.0 (the 'License') and you may not use these files
+ except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop
+ files are distributed onan 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Paweł Marzec <pawel.marzec@modusbox.com>
+ --------------
+ ******/
+import { thirdparty as tpAPI } from '@mojaloop/api-snippets'
+import { validate } from '~/domain/fido-credential'
+
+describe('fido-credential', () => {
+  describe('validate', () => {
+    it('should accept valid credential', async () => {
```

Comment:
Can you also add test cases for invalid credentials? Even just 2 to start with:
1. `type !== 'public-key'`
2. Unsupported attestation format

Reply:
will do

Reply:
tests done